src/devices/src/virtio/block/io/async_io.rs

```diff
@@ -180,27 +180,31 @@ impl<T> AsyncFileEngine<T> {
         self.ring.submit().map(|_| ()).map_err(Error::IoUring)
     }
 
-    pub fn drain(&mut self, flush: bool) -> Result<(), Error> {
-        self.drain_submission_queue()?;
+    pub fn drain(&mut self, flush: bool, discard: bool) -> Result<(), Error> {
```

Comment:
I'd rename this param to `discard_cqes`

Reply:
Hm I think you forgot to rename this

Reply:
Should be renamed: https://github.com/firecracker-microvm/firecracker/blob/81930e4a5599956a1c45a4c4f1e429a7831d6ae3/src/devices/src/virtio/block/io/async_io.rs#L184